apps/api/src/app/auth/services/passport/jwt.strategy.ts

```diff
@@ -10,16 +11,31 @@ export class JwtStrategy extends PassportStrategy(Strategy) {
     super({
       jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
       secretOrKey: process.env.JWT_SECRET,
+      passReqToCallback: true,
     });
   }
 
   @Instrument()
-  async validate(payload: UserSessionData) {
-    const user = await this.authService.validateUser(payload);
+  async validate(req: http.IncomingMessage, session: UserSessionData) {
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
🤓 **nitpick (non-blocking):** ‏We should probably standardize on using a single type for the request object. It appears that Request from the global namespace is the most commonly used type.